wardrobe/tradable_kitties/src/tests.rs

```diff
@@ -0,0 +1,864 @@
+//! Tests for the Crypto Kitties Piece
+
+use super::*;
+use kitties::DadKittyStatus;
+use kitties::KittyDNA;
+use kitties::Parent;
+use sp_runtime::testing::H256;
+
+/// A bogus data type used in tests for type validation
+#[derive(Encode, Decode)]
+struct Bogus;
+
+impl UtxoData for Bogus {
+    const TYPE_ID: [u8; 4] = *b"bogs";
+}
```

Comment:
This already exists for you. You don't need to re-create it.

https://github.com/Off-Narrative-Labs/Tuxedo/blob/1ad23430126711d6fca8899e4f00306e76bf98f8/tuxedo-core/src/dynamic_typing.rs#L138-L149

Reply:
fixed this.